gef-ui/server/controllers/amendments/helpers/create-amendment-reference-number.helper.ts

```diff
@@ -1,14 +1,14 @@
-import { PORTAL_AMENDMENT_STATUS } from '@ukef/dtfs2-common';
+import { PORTAL_AMENDMENT_STATUS, FacilityAllTypeAmendmentWithUkefId, TFM_AMENDMENT_STATUS, createReferenceNumber } from '@ukef/dtfs2-common';
 import api from '../../../services/api';
 /**
  * @param dealId - the deal id
  * @param facilityId - the facility id
  * @param userToken - the user token
  * @returns the reference number
  */
-export const createReferenceNumber = async (dealId: string, facilityId: string, userToken: string): Promise<string> => {
+export const createReferenceNumberHelper = async (dealId: string, facilityId: string, userToken: string): Promise<string> => {
```

Comment:
```suggestion
export const createAmendmentReferenceNumber = async (dealId: string, facilityId: string, userToken: string): Promise<string> => {
```

I don't think we need helper here in the const name as we don't for many others

Reply:
@zainUKEF the reason I changed this name is because I added a createReferenceNumber function into libs/common (in order to use it when I create a referenceNumber on for tfm amendments). . I call this function createReferenceNumber into this helper